src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/IVSTypeScriptEditorInlineRenameService.cs

```diff
@@ -12,8 +13,17 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api
     /// <summary>
     /// Language service that allows a language to participate in the editor's inline rename feature.
     /// </summary>
+    [Obsolete]
```

Comment:
Tracking bug for the removal of this?

Reply:
Populating the "Obsolete" text with something like "please use VSTypeScriptEditorInlineRenameServiceImplementation instead" would also be good.